packages/elements/src/internals/machines/sign-in/machines/verification.machine.ts

```diff
@@ -53,6 +54,38 @@ const SignInVerificationMachine = setup({
     determineStartingFactor: () => {
       throw new ClerkElementsRuntimeError('Action `determineStartingFactor` must be overridden');
     },
+    validateConfiguredStrategies: ({ context }) => {
+      const clerk = context.parent.getSnapshot().context.clerk;
+
+      if (clerk.__unstable__environment?.isProduction()) {
+        return;
+      }
+
+      if (
+        !clerk.client.signIn.supportedFirstFactors.every(factor =>
+          context.registeredStrategies.has(factor.strategy as unknown as SignInFactor),
+        )
+      ) {
+        console.warn(
+          `Clerk: Your instance is configured to support strategies: ${clerk.client.signIn.supportedFirstFactors
+            .map(f => f.strategy)
+            .join(', ')}, but the rendered strategies are: ${[...context.registeredStrategies]
+            .map(s => s)
+            .join(
+              ', ',
+            )}. Before deploying your app, make sure to render a <Strategy> component for each supported strategy. For more information, visit the documentation: <link>`,
```

Comment:
TODO: Add link to docs

Reply:
Most likely: `https://clerk.com/docs/elements/reference/sign-in#strategy`

Reply:
https://github.com/clerk/javascript/pull/3165/commits/ef491eefa328185725d77ef84b3f968f5519fa33